relativedeltafield/__init__.py

```diff
@@ -87,13 +148,15 @@ class RelativeDeltaField(models.Field):
 					 "ISO8601 interval format.")
 	}
 	description = _("RelativeDelta")
-
+	form_class = RelativeDeltaFormField
+	descriptor_class = RelativeDeltaDescriptor
 
 	def db_type(self, connection):
-		if connection.vendor == 'postgresql':
+		if connection.vendor == 'postgresql3':
 			return 'interval'
 		else:
-			raise ValueError(_('RelativeDeltaField only supports PostgreSQL for storage'))
+			return 'varchar(27)'
```

Comment:
Let's restore the original code.  I have no intention of supporting databases other than Postgresql. This project was specifically intended to support `interval` fields on Postgres. There are no tests for other databases, and I would hate it if the above change would silently cause my fields to be silently converted to varchars.

Reply:
This was a proposal: we could initially fall back to a varchar with other databases and if they provide a field type similar to the Postgres interval we can think of implementing it later?